### ISSUE TITLE
Clicking on panel label does not work after AJAX request

### ISSUE THREAD

**Issue by JanTvrdik** [nette/tracy#177]
Latest Tracy, `examples/ajax.php`, after clicking on "AJAX request" button, you can no longer change info or dump panel mode from peek to floating.

**Comment by JanTvrdik** [comment]
Thanks.
